corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -531,9 +531,11 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
                 }).addTo(addressMap);
 
                 const addressIndex = _.findIndex(this.styles, function (style) { return style.displayFormat === constants.FORMAT_ADDRESS; });
+                const popupIndex = _.findIndex(this.styles, function (style) { return style.displayFormat === constants.FORMAT_ADDRESS_POPUP; });
                 L.mapbox.accessToken = token;
+                md = window.markdownit();
```

Comment:
Does this need a declaration (i.e. `const`) or does that break things because it's already defined elsewhere?

Reply:
Having just looked at https://github.com/dimagi/commcare-hq/pull/33155, I think this could maybe make use of the `cloudcareUtils.renderMarkdown` added there.